### ISSUE TITLE
Deployment of field-campaigner in development mode

### ISSUE THREAD

**Issue by Dragneel7** [hotosm/field-campaigner#417]
While setting up field-campaigner on my system, I faced the following issues : 
 
**Docker**
run commands 
` make build`
  `make run`  
 are for the production mode.
 I think it should be specified [here](https://github.com/hotosm/field-campaigner/wiki/Deployment) that for running in development mode, one should run 
                          `make  build-dev`
                          `make run-dev`

**Manual**
It should be mentioned that the app is built in python3 and should use pip3 instead of pip(incase python2 is set as default in the user's system)
and, to deploy the app in development mode run: 
`pip3 install -r requirements-dev.txt`
and uncomment the line
`# -r requirements.txt`   [here](https://github.com/hotosm/field-campaigner/blob/develop/deployment/docker/requirements-dev.txt)
Doing this worked for me.

@meomancer @smit1678 what do you think about it?

**Comment by smit1678** [comment]
@Dragneel7 Yeah I've run into this as well. I created a `docker-compose-local.yml` file that has the env variables pre-filled and used that instead of the Make file. I think if we can clean up the makefile and offer some additional instructions as you say, that'd be great.

**Comment by Dragneel7** [comment]
Sure thing we can discuss upon that.
